### PR TITLE
Expose model_version in scanners API response

### PR DESCRIPTION
**Pull request by willdurand**
Fixes https://github.com/mozilla/addons-server/issues/14931

### CODE REVIEW

File: docs/topics/api/scanners.rst [hunk 1]
```diff
@@ -28,3 +28,4 @@ This endpoint returns a list of labelled scanner results.
     :>json string label: Either ``good`` or ``bad``.
     :>json object results: The scanner (raw) results.
     :>json string created: The date the result was created, formatted with `this format <http://ecma-international.org/ecma-262/5.1/#sec-15.9.1.15>`_.
+    :>json string|null model_version: The model version when applicable, ``null`` otherwise.
```
Comment (eviljeff) [inline docs/topics/api/scanners.rst:31]: this sentence doesn't really mean anything to someone who doesn't understand the whole scanner ecosystem.  What's a model, what does it's version mean, and when is it applicable?

File: src/olympia/scanners/tasks.py [hunk 2]
```diff
@@ -478,13 +478,13 @@ def call_mad_api(all_results, upload_pk):
             score=data.get('ensemble', default_score),
```
Comment (eviljeff) [inline src/olympia/scanners/tasks.py:478]: so, you get the `model_version` from `MAD_API_URL` results and add it the `ScannerResult(scanner=CUSTOMS)` record, but not the `ScannerResult(scanner=MAD)` record?

File: docs/topics/api/scanners.rst (hunk 1 above)
Comment (willdurand) [reply docs/topics/api/scanners.rst:31]: Vagueness is a feature here.

File: src/olympia/scanners/tasks.py (hunk 2 above)
Comment (willdurand) [reply src/olympia/scanners/tasks.py:478]: Yes. This `model_version` is specific to customs results. We still have the raw response stored on the mad results, though.

**Pull request merged**